readme.md

```diff
@@ -65,6 +72,7 @@ Describe the purpose of views and models in FE JS.
 
 Your Answer:
 ```text
+Views and models work together on the FE to display backend data to the FE.
```

Comment:
FE models use AJAX to stay in sync with BE models, FE views take the data from the models to render as HTML